index.html

```diff
@@ -0,0 +1,123 @@
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#main">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+            </ul>
+
+        </nav>
+        <nav aria-label="Menu" class="desktop-menu">
+            <a class="logo" href="index.html"></a>
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#main">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><button type="button">Campaign</button></li>
+            </ul>
+        </nav>
+    </header>
+    <section id="banner" class="banner">
+        <video id="background-video" muted loop autoplay playsinline preload="auto" poster="Images/banner_Moment.jpg">
+            <source type="video/mp4" src="Images/banner.mp4" />
+        </video>
+        <div class="banner-page">
+            <i class="menu-button"></i>
+            <div>
+                <h2>Welcome to the</h2>
+                <h1>12th Software Development<br>Summit 2023</h1>
+            </div>
+            <p class="brief-description-banner"> Lorem Ipsum has been the industry's standard dummy text ever since the
+                1500s, when an unknown
+                printer took a galley of type and scrambled it 1960s.</p>
+            <div class="date-location">
+                <h2>2023.04.22(SAT) ~ 23(SUN)</h2>
+                <p>@XPO, Ministry of Inovation & Technology-Ethiopia</p>
+            </div>
+        </div>
+    </section>
+    <section id="main">
+        <h2 class="section-headline">Main Program</h2>
+        <div class="main-container">
+            <div class="program">
+                <i class="i-lecture"></i>
+                <h3>Lecture</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum reprehenderit.</p>
+            </div>
+            <div class="program">
+                <i class="i-exhibition"></i>
+                <h3>Exhibition</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum reprehenderit.</p>
+            </div>
+            <div class="program">
+                <i class="i-forum"></i>
+                <h3>Forum</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum reprehenderit.</p>
+            </div>
+            <div class="program">
+                <i class="i-workshop"></i>
+                <h3>Workshop</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum reprehenderit.</p>
+            </div>
+            <div class="program">
+                <i class="i-ignite"></i>
+                <h3>Lecture</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum reprehenderit.</p>
```

Comment:
**OPTIONAL** - You have done well so far, The text in this section are overflowing for all individual cards, Please fix this 🙏.  Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
![wgwgaeg](https://user-images.githubusercontent.com/73607512/233167338-9bffdc48-82cd-45f2-a4d9-9e7ac1eed5b3.PNG)